apache-rat-core/src/main/resources/org/apache/rat/plain-rat.xsl

```diff
@@ -53,42 +53,58 @@ Files with unapproved licenses:
 </xsl:if>
 <xsl:if test="descendant::resource[type/@name='archive']">
 Archives:
-<xsl:for-each select='descendant::resource[type/@name="archive"]'>
+<xsl:for-each select='descendant::resource[@type="archive"]'>
  + <xsl:value-of select='@name'/>
  <xsl:text>
  </xsl:text>
  </xsl:for-each>
 </xsl:if>
+<xsl:text>
 *****************************************************
-  Files with Apache License headers will be marked AL
-  Binary files (which do not require any license headers) will be marked B
-  Compressed archives will be marked A
-  Notices, licenses etc. will be marked N
+  Documents with unapproved licenses will start with a '!'
+  The next character identifies the document type.
+   
+   char         type
+    a       Archive file
```

Comment:
Should we use uppercase chars to identify the document's type to be more compliant with current RAT reports?